jenkins_jobs/update_versions_readme/script.sh

```diff
@@ -0,0 +1,9 @@
+python host-os.py \
+  --verbose \
+  update-versions-readme \
+    --build-versions-repository-url ${VERSIONS_REPO_URL} \
+    --build-version ${VERSIONS_REPO_COMMIT} \
+    --updater-name "${GITHUB_BOT_NAME}" \
+    --updater-email "${GITHUB_BOT_EMAIL}" \
+    --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
```

Comment:
Because this is the way it is done in other scripts.

Refer to https://github.com/open-power-host-os/infrastructure/blob/master/jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh, for instance.

I'm being consistent with other scripts.